From 9f84b1a71df2f7ba035db8f5af109a1d4120fa58 Mon Sep 17 00:00:00 2001
From: Stanislav Ochotnicky <sochotnicky@redhat.com>
Date: Mon, 17 May 2010 13:48:31 +0200
Subject: [PATCH 02/10] Remove version dependencies

this is to remove (unneeded) versioned dependencies until we upgrade maven
---
 pom.xml |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/pom.xml b/pom.xml
index 977ca83..ae0d95e 100644
--- a/pom.xml
+++ b/pom.xml
@@ -95,7 +95,6 @@ target platform and specify -Dntdll_target=msbuild on the mvn command line.
       </plugin>
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.5</version>
         <configuration>
           <workingDirectory>tests</workingDirectory>
           <reportFormat>plain</reportFormat>
@@ -149,7 +148,6 @@ target platform and specify -Dntdll_target=msbuild on the mvn command line.
       </plugin>
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>2.1</version>
         <configuration>
           <source>1.3</source>
           <target>1.1</target>
@@ -332,7 +330,6 @@ target platform and specify -Dntdll_target=msbuild on the mvn command line.
       </plugin>
       <plugin>
         <artifactId>maven-javadoc-plugin</artifactId>
-        <version>2.6.1</version>
         <configuration>
           <docfilessubdirs>true</docfilessubdirs>
           <encoding>UTF-8</encoding>
-- 
1.6.6.1

